src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -198,6 +198,7 @@ class BaseExpensiTextInput extends Component {
                                 onBlur={this.onBlur}
                                 onChangeText={this.setValue}
                                 onPressOut={this.props.onPress}
+                                translateX={this.props.translateX}
```

Comment:
@Jag96 Just want to know where did you get to know about this prop? It's not in docs...

Reply:
I think it's a deprecated view prop: https://reactnative.dev/docs/transforms#decomposedmatrix-rotation-scalex-scaley-transformmatrix-translatex-translatey

Reply:
I only knew about it because we've used it in other parts of the app, I wasn't aware it was deprecated

Reply:
Interesting.